README.md

```diff
@@ -1,20 +1,30 @@
 # orca
+
 Orca manages people and activity subscriptions.
 
 ## Development
 
 ```
 docker-compose up app
 ```
+
 or open with VS Code Dev Container Extension
 
+### Webpack Dev Server
+
+Inside the app container start
+
+```
+bin/webpack-dev-server
```

Comment:
I'd recommend running this in a separate container. We do it similarly in https://github.com/hitobito/development/blob/master/docker-compose.yml, I'd be happy to help out with that if you need support.

Reply:
@carlobeltrame Ich hatte es bisher auch in einem separaten Container, sehe aber die Benefits nicht mehr so wirklich (ausser dem manuellen Starten). Auf der Anderen Seite sind es zwei identische Container die zu managen sind; Webpacker braucht das ganze bundle und die node_modules. Volumes sharen fand ich problematisch weil es dann zu Race Conditions beim aufsetzen kommen kann. Ihr habt es auch so, dass die jeweiligen container separat bundlen. Kein Problem?

Mich würde deine Meinung und Erfahrungen sehr interessieren! Danke fürs melden!

Reply:
Es ist wohl eine philosophische Frage. Ich ziehe es separat vor weil bei Docker generell empfohlen wird, pro laufenden Prozess ein Container zu haben. Der Webpack-Dev-Server ist ein Musterbeispiel dafür, da er relativ unabhängig vom Rest der Applikation starten und stoppen kann, und die anderen Services weiterlaufen können. Diese Art zu arbeiten erlaubt es mir auch, unabhängig von verwendeter Sprache und Framework denselben Befehl `docker-compose up` ohne weiteres Wissen zu verwenden, um das ganze Projekt zu starten. Den Overhead wegen dem doppelt geführten Bundle habe ich bei hitobito noch nie wirklich bewusst bemerkt, vor allem mit YAML-Vererbung ist das auch im docker-compose.yml relativ schmerzfrei zu unterhalten. Es sind auch "nur" die fertigen Container die doppelt sind, das Image ist ja dasselbe.

Ich arbeite aber auch auf Linux, und hauptsächlich ausserhalb der Container. Wenn du mit VSCode direkt in einem Container drin arbeitest oder immer eine Konsole im Container offen hast und alles dort drüber machst, hat es natürlich Vorteile, wenn Webpacker gleich auch dort läuft. Wo mir von aussen `docker-compose ps` einen Überblick über den Status der Prozesse gibt verwendest du wohl lieber einfach `ps` im Container drin. Und auf Mac oder Windows könnte es zugegebenermassen auch aufwändiger sein, denselben Code in zwei Container zu synchronisieren statt nur in einen.

Reply:
Danke für die Anstösse, ich habe den Container wieder dazugenommen weil ich auch finde das es mehr der Docker-Philosophie entspricht.